website/src/_posts/2021-12-2.0-2.3.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Life after 2.0: new Audio plugin, Status Bar improvements" 
+date: 2021-12-15
+author: 
+  - aduh95
+  - murderlon
+  - arturi
+image: https://uppy.io/images/blog/2.0/uppy-2-0-cover.jpg
+published: true
+---
+
+After the 2.0 release we’ve been busy refactoring, updating dependencies, moving from npm to Yarn 3 and much-needed housekeeping. We’ve also released a new Audio plugin, improved Status Bar and Unsplash
+
+<!--more-->
+
+## Audio plugin
+
+Uppy now has a new `@uppy/audio` plugin for recording audio memos, messages and sounds. Check it out [on the demo page](https://uppy.io/examples/dashboard/).
+
+<video alt="Audio plugin demo" muted autoplay loop>
+  <source src="/images/blog/2.0-2.3/audio-demo.mp4" type="video/mp4">
+  Your browser does not support the video tag: https://uppy.io/images/blog/2.0-2.3/audio-demo.mp4
+</video>
+
+## Status bar error state improvements
+
+Status bar plugin would get confused about upload errors in Uppy, and we’ve addressed that:
+
+* Error state is shown if only _one_ failed file instead of _all_ failed files. Used to be stuck in the uploading state when a file failed.
+* Added “x of x files uploaded” below “Upload failed” for extra context.
+* Improved the error details button styling in the statusbar and the file info card.
+* Status bar state is set to complete if the user manually removes the failed files.
+
+![status bar improvements screenshot](/images/blog/2.0-2.3/status-bar-improvements.jpg)
+
+PR [#3299](https://github.com/transloadit/uppy/pull/3299).
```

Comment:
Since when do we add PRs in the blogpost?

Reply:
I think that's new indeed, although that looks fine to me. Do you have any thoughts on it yourself?

Reply:
https://uppy.io/blog/2021/04/1.27/#Dashboard-Dynamic-metaFields we do sometimes, usually when I think there might be interesting details in there.

I think the feedback is becoming a bit harsh? 🙈

Reply:
> I think the feedback is becoming a bit harsh? 🙈

Pardon me, couldn't have phrased that better for sure. I think PR links could be added, yes. Was mostly curious if that's a new thing or not